src/Riak/Api/Http.php

```diff
@@ -12,7 +12,7 @@
 use Basho\Riak\DataType\Hll;
 use Basho\Riak\Location;
 use Basho\Riak\Node;
-use Basho\Riak\Object;
+use Basho\Riak\RObject as RObject;
```

Comment:
Sadly I am not sure if any of the people with merging permission will ever look at this PR but, for what is worth I would like to thank you for this contribution. I created a similar PR previously but it was never accepted due to the upcoming redesign of the library.

Interestingly enough even during the design they didn't address the issue apparently.

Anyway, I would discourage the use of an alias here considering the alias doesn't add anything. I'd say just reference the object as is. I wondered myself if there is a better naming we could use in this case to refer to a Riak Object in a clearer way. I like when names match the domain language in the most unambiguous way possible. Thoughts?

Reply:
It has been a long time since I looked at this repo. I no longer have write access and haven't contributed to any PHP projects in almost 10 years since shifting my career toward Go.
![image](https://user-images.githubusercontent.com/20638/213706855-1224dafe-9f60-44e8-b298-a1b20c9e5a43.png)

I suspect the only person with write access is @russelldb, the last remaining member of the Basho org.
![image](https://user-images.githubusercontent.com/20638/213707125-b1a1f226-8a01-4274-adf9-2293aa5f1a8d.png)